packages/components/dialog/src/dialog.scss

```diff
@@ -126,18 +109,18 @@ slot[name='footer'] {
 }
 
 dialog[closing] {
-  animation: slide-down var(--_closing-duration) ease-out;
+  animation: slide-down var(--_closing-duration) both ease-out;
 
   &::backdrop {
-    animation: backdrop-hide var(--_closing-duration) ease-out;
+    animation: backdrop-hide var(--_closing-duration) both ease-out;
   }
 }
 
 dialog[open]:not([closing]) {
-  animation: slide-in-up var(--_opening-duration) ease-out;
+  animation: slide-in-up var(--_opening-duration) both ease-out;
 
   &::backdrop {
```

Comment:
it looks like the backdrop is not animating when closing the dialog with `Escape` key?

Reply:
Good catch, fixed!